indigo_app/templates/indigo_api/work_new_batch.html

```diff
@@ -184,22 +184,19 @@ <h4 class="card-header">
       <div class="card-body">
         Instructions:
         <ul>
-          <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1JJKal7IFjFKKojsHIJAW12elDUNIwAuMrCV4sJ1jUOc">Work info template</a>: 'File' | 'Make a copy…'</li>
-          <li>Make it accessible: 'SHARE'  | 'Get shareable link' (top-right corner)</li>
-          <li>Watch <a href="https://www.youtube.com/watch?v=tfNm_PtZ7JQ&index=2&list=PL7MJ_sFHs953q_23TdDhnvQWqxAkrc672">this video</a> for instructions on how to populate the spreadsheet</li>
-          <li>Choose a country from the drop-down</li>
-          <li>Paste the url for your spreadsheet into the box below that</li>
-          <li>Click on 'Import works'!</li>
+          <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1JJKal7IFjFKKojsHIJAW12elDUNIwAuMrCV4sJ1jUOc" target="_blank">Work info template</a>: 'File | Make a copy'.</li>
+          <li>Record the Spreadsheet URL under <a href="{% url 'place_settings' place=place.place_code %}" target="_blank">Settings</a>.</li>
+          {% if bulk_creator.is_gsheets_enabled %}
+            <li>Share the spreadsheet with (give Edit rights to) {{ share_with }}.</li>
```

Comment:
I think "(give Edit rights to)" is confusing. It technically doesn't need edit rights, and "give edit rights" is also not a term that Google Drive uses, whereas "share with" is. I think we can safely drop "(give Edit rights to)".